src/gatling/scala/uk/gov/hmcts/reform/sendletter/EmbeddedPdfsSimulation.scala

```diff
@@ -0,0 +1,29 @@
+package uk.gov.hmcts.reform.sendletter
+
+import com.typesafe.config.{Config, ConfigFactory}
+import io.gatling.core.Predef._
+import io.gatling.http.Predef._
+import uk.gov.hmcts.reform.sendletter.actions.{LettersService, S2s}
+
+import scala.concurrent.duration._
```

Comment:
i believe from the package code is only using `IntDuration` or smth - don't exactly remember the name of implicit converter